admin/conference-template/process/commentProcess.template

```diff
@@ -21,15 +21,15 @@ function(){
       if(!selfComment){
         if(replytoNote.id == origNote.id){
           author_mail = {
-            "groups": origNote.content.authorids,
-            "subject": "Your submission to "+CONFERENCEPHRASE+" has received a comment",
-            "message": "Your submission to "+CONFERENCEPHRASE+" has received a comment.\n\nComment title: " + note.content.title + "\n\nComment: " + note.content.comment + "\n\nTo view the comment, click here: " + baseUrl+"/forum?id=" + note.forum +'&noteId='+note.id
+            'groups': origNote.content.authorids,
```

Comment:
Why the change from double quotes to single quotes?

Reply:
I remember @zbialecki telling me that single quotes are preferred in javascript. is that right Zach?

Reply:
This is correct, it's mainly a matter of style preference but we standardized on single quotes in the OpenReview codebase. Also object properties like group, subject and message do not need any quotes.